### ISSUE TITLE
[Feature Request] Export previewed images to ROM's folder

### ISSUE THREAD

**Issue by stonie08** [doZennn/steam-rom-manager#329]
The preview screen allows to select a new cover image for each ROM. However if SRM is launched on different machines for the same ROM library the images selection has to be done once for each machine. An easy way to work around this issue would be to specify a pattern for local images. However, there is currently no easy way (that I'm aware of) to export the images one selected in the preview screen and save all of them with a file-naming pattern into the ROM's folder(s) for a the next parser run to grab. Currently only single images can be saved manually, which would make this a tedious process. The file naming pattern should allow to save the images according to a template.

An export of the previewed images would allow to save the state of selected artwork into the ROM's library in a filesystem based way and allow it being easily replicated on another device by e.g. a file sync application of your choice. SRM could then be used on any of the machines to manage the artworks that are then available on the other machines.

**Comment by MattMckenzy** [comment]
Hi @stonie08, I've added an export/import feature to my fork here: https://github.com/MattMckenzy/steam-rom-manager.
You can try to grab it and build it if you need it! Let me know if you need any assistance.

**Comment by stonie08** [comment]
@MattMckenzy Thanks for the implementation! I just packaged it to a flatpak and checked it out. It's almost exactly what I had in mind. Some feedback from testing it:
- It currently seems to save the same banner image for each category (poster, grid,..)
- Both on Win and Linux is seems to have an issue if there's no image available resulting in a "Could not save slections package folder: Cannot read property 'ImageUrl' of undefined", which also aborts the export instead of skipping it. (The slections typo is in the error message)
- Because it didn't save alle the selections or something, upon "import and save selection" I got a "Could not import selections ENDENT: no such file or directory, open '[....]/Selections.json"
- On the feature-side: I wish there was an option to save it to the specific ROMs directory. I have each ROM in a separate directory, so just placing a file named "banner.png in there would be ideal for me. This would probably make it easy to to remove the need for the "import" as the parser's import image regex could be used, so less duplicate functionality.
